lib/htmltoword/xslt/base.xslt

```diff
@@ -204,6 +204,12 @@
         <xsl:when test="self::a[starts-with(@href, 'http://') or starts-with(@href, 'https://')]">
           <xsl:call-template name="link" />
         </xsl:when>
+        <xsl:when test="self::img">
```

Comment:
I think this is not working, also if we support `<a...><img...></a>` this should also be in the link template on `links.xslt`